shared/locales/en/options.json

```diff
@@ -0,0 +1,90 @@
+{
+    "smartling": {
+        "string_format" : "icu",
+        "translate_paths": [{
+            "path": "*/title",
+            "key": "{*}/title",
+            "instruction": "*/note"
+        }]
+    },
+    "optionsHeader": {
+        "title": "DuckDuckGo Options",
+        "note": "Header for options (settings) page"
+    },
+    "optionsSubHeader": {
+        "title": "Search and browse the web without being tracked.",
+        "note": "We allow the user to browser the web without being followed by trackers"
+    },
+    "optionsDesc": {
+        "title": "DuckDuckGo protects your privacy online with\nprivate search,\ntracker blocking,\nand site encryption.",
+        "note": "This describes how we protect the user from being tracked online"
+    },
+    "shareFeedback": {
+        "title": "Share feedback",
+        "note": "Click here to share your opinions about this product"
+    },
+    "reportBrokenSite": {
+        "title": "Report broken site",
+        "note": "Click here to report an issue with a specific site you are trying to use"
+    },
+    "showEmbeddedTweets": {
+        "title": "Show embedded Tweets",
+        "note": "Whether the user wants to see Tweets embedded in various webpages or should they be hidden util clicked on"
+    },
+    "globalPrivacyControlAbbr": {
+        "title": "Global Privacy Control (GPC)",
+        "note": "Global Privacy Control is the name of the feature that lets users set a preference to not have their information sold"
+    },
+    "globalPrivacyControlDesc": {
+        "title": "Your data shouldn't be for sale. At DuckDuckGo, we agree.\nActivate the \"Global Privacy Control\" (GPC) settings and we'll\nsignal to websites your preference to:",
+        "note": "This describes the GPC feature, which lets the user set a preference to not have their personal information sold by websites to other parties"
+    },
+    "notSellYourPersonalData": {
+        "title": "Not sell your personal data.",
+        "note": "Part of the GPC description, this communicates that the user would signal to not SELL their personal data"
+    },
+    "limitSharingOfPersonalData": {
+        "title": "Limit sharing of your personal data to other companies.",
+        "note": "Part of the GPC description, this communicates that the user would signal to not SHARE their personal data"
+    },
+    "globalPrivacyControlDisclaimer": {
+        "title": "<b>Since Global Privacy Control (GPC) is a new standard,\nmost websites won't recognize it yet, but we're working hard\nto ensure it becomes accepted worldwide.</b> However, websites are only required to act on the signal to the\nextent applicable laws compel them to do so.&nbsp;",
```

Comment:
Is the `&nbsp;` needed or there by mistake?

Reply:
Its there to provide a space in the UI before the "Learn More" link, otherwise its sitting right on that period.